src/App.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
-import './App.css';
+// eslint-disable-next-line sort-imports
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import SideNavbar from 'components/SideNavbar/SideNavbar';
+import chatPage from 'pages/chatPage';
+import historyPage from 'pages/historyPage';
+import keywordPage from 'pages/keywordPage';
+import myPage from 'pages/myPage';
 
-function App() {
-  return (
-    <div>
-      Loading
-    </div>
-  );
-}
+const App = () => (
+  <div className="Main">
+    <SideNavbar />
+    <Routes>
+      <Route path="/history" component={historyPage} exact />
+      <Route path="/chat" component={chatPage} exact />
+      <Route path="/keyword" component={keywordPage} exact />
+      <Route path="/mypage" component={myPage} exact />
```

Comment:
중복되는 path 가 없어 `exact` 속성을 굳이 안써도 좋을것 같습니다.

```suggestion
      <Route path="/history" component={historyPage} />
      <Route path="/chat" component={chatPage} />
      <Route path="/keyword" component={keywordPage} />
      <Route path="/mypage" component={myPage} />
```

Reply:
cbfe814